pom.xml

```diff
@@ -34,20 +34,20 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding> 
 
     <!--  Need Jersey+Jetty for testing; deps from DropWizard 1.2.9 -->
-    <version.jersey>2.25.1</version.jersey>
+    <version.jersey>3.0.0-M1</version.jersey>
     <!-- wrt CVE-2019-10247 -->
     <version.jetty>9.4.17.v20190418</version.jetty>
```

Comment:
9.4.29.v20200521

Reply:
Sure I can update this, I just didn't think my commits should touch other dependency not strictly relevant to the pull request.